fix/test-user-validation (#2360)
mercadopago/devsite-docs: guides/adobe-commerce/integration-test.pt.md

![Login](/images/adobe-commerce/test-login.gif)

 > WARNING
 >
 > Importante
 >

>  > Se, ao fazer login com uma conta de teste ou navegar pelas seções de Suas integrações solicitarem a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste).

Reviewer: ```suggestion
 > Se, ao fazer login com uma conta de teste ou navegar pela seção "Suas integrações", for solicitada a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste).
```

>  > Se, ao fazer login com uma conta de teste ou navegar pelas seções de Suas integrações solicitarem a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste).

Reviewer: ```suggestion
 > Se, ao fazer login com uma conta de teste ou navegar pelas seções de Suas integrações, for solicitada a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste).
```

>  > Se, ao fazer login com uma conta de teste ou navegar pelas seções do Painel do desenvolvedor, solicitarem a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste).

Author: ```suggestion
 > Se, ao fazer login com uma conta de teste ou navegar pelas seções de Suas integrações solicitarem a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste).
```

![Painel](/images/adobe-commerce/test-adobe-commerce.png)

11. Clique em **Save Config**.
12. Abra uma nova janela anônima e faça login no Mercado Pago usando a conta de teste do comprador criada no passo 3. 

>     > WARNING

Reviewer 2: ```suggestion
> WARNING
```

>     >

Reviewer 2: ```suggestion
>
```

>     > Importante

Reviewer 2: ```suggestion
> Importante
```

>     >

Reviewer 2: ```suggestion
>
```

>     > Se, ao fazer login com uma conta de teste ou navegar pelas seções do Painel do desenvolvedor, solicitarem a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste). 

Reviewer 2: ```suggestion
> Se, ao fazer login com uma conta de teste ou navegar pelas seções do Painel do desenvolvedor, solicitarem a autenticação por e-mail, acesse nossa documentação para saber [como validar o login em contas teste](/developers/pt/docs/adobe-commerce/additional-content/your-integrations/test/accounts#bookmark_validar_login_com_usuarios_teste). 
```
